packages/ecs-agent/ecs.service

```diff
@@ -0,0 +1,34 @@
+[Unit]
+Description=Amazon Elastic Container Service - container agent
+Documentation=https://aws.amazon.com/documentation/ecs/
+Requires=docker.service
+After=docker.service configured.target
+
+[Service]
+Type=simple
+Restart=on-failure
+RestartPreventExitStatus=5
+RestartSec=1s
+EnvironmentFile=-/etc/ecs/ecs.config
+Environment=ECS_LOGFILE=/var/log/ecs/ecs-agent.log ECS_LOGLEVEL=debug
+Environment=ECS_CHECKPOINT=true
+ExecStartPre=mkdir -p /var/log/ecs /var/lib/ecs
+ExecStartPre=/sbin/iptables -t nat -A PREROUTING -d 169.254.170.2/32 \
+ -p tcp -m tcp --dport 80 -j DNAT --to-destination 127.0.0.1:51679
+ExecStartPre=/sbin/iptables -t nat -A OUTPUT -d 169.254.170.2/32 \
+ -p tcp -m tcp --dport 80 -j REDIRECT --to-ports 51679
+ExecStartPre=/sbin/iptables -t filter -I INPUT --dst 127.0.0.0/8 ! \
+ --src 127.0.0.0/8 -m conntrack ! --ctstate RELATED,ESTABLISHED,DNAT -j DROP
+ExecStartPre=/sbin/sysctl -w net.ipv4.conf.all.route_localnet=1
+ExecStart=/usr/bin/amazon-ecs-agent
+ExecStopPost=-/sbin/iptables -t nat -D PREROUTING -d 169.254.170.2/32 \
+ -p tcp -m tcp --dport 80 -j DNAT --to-destination 127.0.0.1:51679
+ExecStopPost=-/sbin/iptables -t nat -D OUTPUT -d 169.254.170.2/32 \
+ -p tcp -m tcp --dport 80 -j REDIRECT --to-ports 51679
+ExecStopPost=-/sbin/iptables -t filter -D INPUT --dst 127.0.0.0/8 ! \
+ --src 127.0.0.0/8  -m conntrack ! --ctstate RELATED,ESTABLISHED,DNAT -j DROP
+ExecStopPost=/bin/sh -c \
```

Comment:
Is the intent to replace this shutdown `sysctl` tuning with something else? We shouldn't have `/bin/sh` available at runtime in the host's rootfs.

Reply:
Yes.  We don't have `/bin/sh` available at runtime, so this command doesn't work today.  From the temporary README.md in the same directory:

> The systemd unit contains many `ExecStartPre`/`ExecStartPost` commands, with little explanation or infrastructure.  One `ExecStopPost` command depends on `/bin/sh`, which is not available in Bottlerocket or compatible with Bottlerocket's goals.

I'm not sure what that replacement will look like yet though.

Reply:
Done